README.md

```diff
@@ -70,19 +72,48 @@ The Kata Containers project is governed according to the ["four opens"](https://
 
 ## Developers
 
-For code contributors, there are currently two roles relevant to project governance:
+For contributors, there are several roles relevant to project governance:
 
 ### Contributor
 
 A Contributor to the Kata Containers project is someone who has had code merged within the last 12 months. Contributors are eligible to vote in the Architecture Committee elections. Contributors have read only access to the Kata Containers repos on GitHub.
 
-### Maintainer
+### Committer
 
-A Maintainer has the ability to merge code into the Kata Containers project. Maintainers are active Contributors and participants in the projects. In order to become a Maintainer, you must be nominated and approved by the established Maintainers. Maintainers have write access to the Kata Containers repos on GitHub.
+Kata Containers Committers (as defined by the [kata-containers-committer team](https://github.com/orgs/kata-containers/teams/kata-containers-committer))
```

Comment:
Note that this link isn't public - you have to be an org member to access it. We may or may not want to keep it here.

Reply:
So I think we'd also publish the proposed committers twice a year under these proposals. I think for github permissions we have to have that team as the primary source of committers though.